_config.php

```diff
@@ -4,11 +4,11 @@
 use Psr\Log\LoggerInterface;
 use SilverStripe\Control\Director;
 use SilverStripe\Core\Injector\Injector;
+use SilverStripe\Core\Environment;
 
-if (!defined('DEBUGBAR_DIR')) {
-    define('DEBUGBAR_DIR', 'debugbar');
+if (!Environment::getEnv('DEBUGBAR_DIR')) {
+    Environment::setEnv('DEBUGBAR_DIR', 'resources/lekoala/silverstripe-debugbar');
```

Comment:
I think that this can be done more nicely with the resource locator...

Reply:
Rather than setting an env var I'd be using the resource locator in all the place you need the path

Reply:
This could be a fun afternoon...